components/proxy/src/main/resources/configuration/default.yml

```diff
@@ -87,3 +87,4 @@ url:
     unwiseCharactersToEncode:
 
 #originRestrictionCookie: yourCookieNameHere
+
```

Comment:
Now the question is if the default values are sensible. Like. `bossThreadCount: 1`? Perhaps that is OK.

How about proxy and admin connectors? Is possible to start styx in HTTPS only?

Reply:
Also, are we able to unset the `backendServiceRegistry` at all?